lib/open_project/backlogs/patches/version_patch.rb

```diff
@@ -58,9 +58,9 @@ def rebuild_positions(project = self.project)
 
         # Add work_packages w/o position to the top of the list and add
         # work_packages, that have a position, at the end
-        stories_wo_position = fixed_issues.find(:all, conditions: { project_id: project, type_id: Story.types, position: nil }, order: 'id')
+        stories_wo_position = fixed_issues.where(project_id: project, type_id: Story.types, position: nil).order('id')
 
-        stories_w_position = fixed_issues.find(:all, conditions: ['project_id = ? AND type_id IN (?) AND position IS NOT NULL', project, Story.types], order: 'COALESCE(position, 0), id')
+        stories_w_position = fixed_issues.where(['project_id = ? AND type_id IN (?) AND position IS NOT NULL', project, Story.types]).order('COALESCE(position, 0), id')
```

Comment:
Line is too long. [168/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.